code/network/multi_options.cpp

```diff
@@ -349,7 +349,7 @@ void multi_options_set_local_defaults(multi_local_options *options)
 	if ( Psnet_connection == NETWORK_CONNECTION_DIALUP ) {
 		options->obj_update_level = OBJ_UPDATE_LOW;
```

Comment:
As much as it seems silly, it's good that we are keeping the top part of this if statement the way it is, in case someone really does have a slow connection.

Reply:
As rare as it might be, there definitely still are people on slow connections. Even if not exactly dial-up, I know of places where entire households will use a single phone's 2G or 3G internet connection routed to other devices via hotspot.